tests/unit/test_global_correction.py

```diff
@@ -1,15 +1,18 @@
 import numpy as np
+import pandas as pd
 import pytest
-from pandas.testing import assert_frame_equal
+from pandas.testing import assert_frame_equal, assert_series_equal
 import tfs
 
-from omc3.correction.constants import VALUE
-from omc3.correction.handler import get_measurement_data
+from omc3.model import manager
+from omc3.correction import response_madx
+from omc3.correction.constants import VALUE, ORBIT_DPP
+from omc3.correction.handler import get_measurement_data, _update_response, _create_corrected_model
 from omc3.global_correction import OPTICS_PARAMS_CHOICES, CORRECTION_DEFAULTS, global_correction_entrypoint as global_correction
 from omc3.optics_measurements.constants import (BETA, DISPERSION, NORM_DISPERSION, F1001, F1010,
                                                 DISPERSION_NAME, BETA_NAME, PHASE_NAME,
                                                 NORM_DISP_NAME, F1010_NAME, TUNE, PHASE,
-                                                F1001_NAME)
+                                                F1001_NAME, EXT, DELTA)
 from omc3.scripts.fake_measurement_from_model import generate as fake_measurement
 from tests.accuracy.test_global_correction import get_skew_params, get_normal_params
```

Comment:
`pd` and `ORBIT_DPP` do not seem to be needed. 
This could also use an auto-format. (ruff, isort, black, etc.)

Reply:
I installed ruff! Hope you like it :)